Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
> invert the array should be done just for bellow Symfony versions.

My thoughts exactly! That way you have a much smaller PR to do in NEXT_MAJOR

> And we should remove the 2 duplicate const colors #FF00FF (MAGENTA/FUCHSIA) and #00FFFF (AQUA/CYAN)

Hmm… wouldn't that be a BC-break? If yes, they should be added back to the flipped array, right ?